tests/NLog.UnitTests/Contexts/MappedDiagnosticsLogicalContextTests.cs

```diff
@@ -45,6 +45,23 @@ public MappedDiagnosticsLogicalContextTests()
         }
 
         [Fact]
+        public void ToSupportObjectVal()
+        {
+            string key = "testKey1";
+            object value = 5;
+
+            MappedDiagnosticsLogicalContext.Set(key, value);
+
+            Assert.Equal(value, MappedDiagnosticsLogicalContext.Get(key));
+        }
```

Comment:
this should be getObject? (see fail on appveyor (AV))

an unit test on MappedDiagnosticsLogicalContext.Get(key) (result should be "5"), would be also nice